presto-iceberg/src/main/java/com/facebook/presto/iceberg/CommitTaskData.java

```diff
@@ -25,19 +26,28 @@ public class CommitTaskData
     private final String path;
     private final long fileSizeInBytes;
     private final MetricsWrapper metrics;
+    private final String partitionSpecJson;
```

Comment:
I think `specId` would be enough, what's your opinion?

Reply:
That made sense to me and I have made changes accordingly. Thanks @hantangwangd!